examples/third_party/repositories.bzl

```diff
@@ -11,8 +11,11 @@ load("//libpng:libpng_repositories.bzl", "libpng_repositories")
 load("//libssh2:libssh2_repositories.bzl", "libssh2_repositories")
 load("//openssl:openssl_repositories.bzl", "openssl_repositories")
 load("//pcre:pcre_repositories.bzl", "pcre_repositories")
+load("//python:python_repositories.bzl", "python_repositories")
 load("//zlib:zlib_repositories.bzl", "zlib_repositories")
 
+# This buildifier suppression is required as "native.register_toolchains" is used in python_repositories
```

Comment:
```suggestion
```

I don't think it's necessary to comment about this. I think the `unnamed-macro` defect is a bit too open and gets flagged on things that shouldn't be an issue. But maybe that's just my own lack of understanding.